vsts/scripts/pullAndTag.sh

```diff
@@ -124,8 +123,25 @@ if [ -n "$TESTINTEGRATIONCASEFILTER" ];then
 		# Always convert filter for runtime images to lower case
 		echo "Runtime image filter is set for $platformFilter with version $platformVersionFilter"
 
+		# Create a local file that consolidates the different runtime image files into one to be read from
+		# Note: we don't write this file to the drop folder as we don't want this file written to for every integration test job
+		sourceFile="$BUILD_SOURCESDIRECTORY/temp/images/runtime-images-acr.txt"
+
+		if [[ ! -f "$sourceFile" ]]; then
+			echo "Creating consolidated runtime image file '$sourceFile'..."
+			touch "$sourceFile"
+		fi
+
```

Comment:
We'll have to do something similar to:
```
		if [[ ! -f "$sourceFile" ]]; then
			echo "Creating consolidated runtime image file '$sourceFile'..."
                        mkdir -p $BUILD_SOURCESDIRECTORY/temp/images/
			touch "$sourceFile"
		fi
```

To fix:
![image](https://github.com/microsoft/Oryx/assets/86327553/7e166261-e1c8-462a-869b-03ef53f14313)

Reply:
https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=8550982&view=logs&j=913ce52b-8ad3-5a93-c0fe-123235d07c67&t=674b07f0-e3ef-53f4-9cc2-286a791ee0ed

Reply:
Thanks for the catch -- for some reason, I just assumed that directory existed, but it makes sense why it doesn't since it's within the scope of the repository code that we clone down.